test_interface_files-extras.cmake.in

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# generated from test_interface_files/test_interface_files-extras.cmake.in
+
+set(@PROJECT_NAME@_MSG_FILES "")
+foreach(msg @msg_files@)
```

Comment:
I would suggest quoting the files (`"@msg_files@"`) to ensure spaces aren't confusing.

*To keep the commit history readable it is probably best to append the change to the commit introducing this.)

Same below.

Reply:
Done.

Reply:
hmm.. I think this was the wrong thing to do. Now I only get the prefix added to the first item in the list. Since `msg_files` is a list (e.g.`"A;B;C"`), reverting the change( removing quotes) looks correct. Items in the list with white space seem to be handled properly.